static/header.less

```diff
@@ -186,7 +186,7 @@
     list-style: none;
     justify-content: center;
     align-items: center;
-    z-index: 999999;
+    z-index: @z-index-top;
```

Comment:
This one doesn’t seem necessary either because the header already has a z-index.